Edit for Customize Email section (#426)
auth0/docs-v2: main/docs/customize/email/smtp-email-providers.mdx

---
title: SMTP Email Provider Overview
sidebarTitle: Overview
description: Configure an external SMTP email provider to send emails in production and to use email templates to customize their appearance and content.
'og:image': https://cdn2.auth0.com/docs/1.14553.0/img/share-image.png
---

Auth0's email-based workflows handle when emails are sent and what the emails contain. To deliver the outgoing mail, you need an SMTP server or service from an external email provider.

> You can configure one email provider which is used for all emails sent from your tenant. We offer three options for this configuration: integration with third-party email providers we support, direct SMTP server conenctions, or custom configuration with Actions.

Reviewer: ```suggestion
You can configure a single email provider to handle all emails sent from your tenant. We offer three options for this configuration: integration with third-party email providers we support, direct SMTP server connections, or custom configuration with Actions.
```

minor typo & tense update

For more information, read [Custom Email Provider Configuration with Actions](/docs/customize/email/smtp-email-providers/custom).

## SMTP Server Configuration

You can configure your tenant by adding SMTP server details directly. This is a good option if:

* We don't support integration with your chosen email provider and you don't need or want to write an Action for configuration.

* You want to set up a dummy SMTP server (like [Debug Mail](https://debugmail.io/), [FakeSMTP](https://nilhcem.github.io/FakeSMTP/), or [smtp4dev](https://github.com/rnwood/smtp4dev)) for testing.

We don't allow certain false domains commonly used during testing. Use real email addresses to avoid disruption or errors.

For instructions, read [Configure an Email Provider with SMTP Server Details](/docs/customize/email/configure-custom-smtp-email-provider).

## The Built-In Email Provider

> The built-in email provider is available by default and requires no configuration. However, itdoes not support production use and has the following limits:

Reviewer: ```suggestion
The built-in email provider is available by default and requires no configuration. However, it does not support production use and has the following limits:
```
minor typo

* You cannot use custom email templates.

> * You cannot customize the **From Address** All emails are sent from `no-reply@auth0user.net`.

Reviewer: ```suggestion
* You cannot customize the **From Address**. All emails are sent from `no-reply@auth0user.net`.
```
Adding punctuation
